src/integrations/admin/indexation-integration.php

```diff
@@ -160,6 +169,8 @@ public function enqueue_scripts() {
 	/**
 	 * Renders the indexation list item.
 	 *
+	 * @codeCoverageIgnore It echos the indexation warning.
```

Comment:
Please write test for this. You can test this by using `expectOutput` or `expectOutputContains`